.github/workflows/actionlint.yml

```diff
@@ -29,27 +29,45 @@ concurrency:
 jobs:
   workflow_syntax:
     if: github.repository_owner == 'Homebrew'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/homebrew/ubuntu22.04:master
     steps:
       - name: Set up Homebrew
         id: setup-homebrew
         uses: Homebrew/actions/setup-homebrew@master
         with:
-          core: false
+          core: true
```

Comment:
Is the core tap needed?

If this change is not intentional then you may also want to relook at the change from `HOMEBREW_REPOSITORY` to `HOMEBREW_TAP_REPOSITORY` below.

Reply:
Oops, thanks

Reply:
Fixed!